Package.swift

```diff
@@ -7,22 +7,27 @@ let package = Package(
     name: "PINOperation",
     products: [
         // Products define the executables and libraries produced by a package, and make them visible to other packages.
-        .library(name: "PINOperation", targets: ["PINOperation"]),
+        .library(
+            name: "PINOperation",
+            targets: ["PINOperation"]),
     ],
     targets: [
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
         // Targets can depend on other targets in this package, and on products in packages which this package depends on.
         .target(
-            name: "PINOperation", 
-            path: "Source", 
-            publicHeadersPath: ".",
+            name: "PINOperation",
+            path: "Source",
+            exclude: ["Info.plist"],
+            resources: [.process("PrivacyInfo.xcprivacy")],
```

Comment:
The Privacy Manifest file needs to be added and referenced within the Xcode project, which is why we are processing the resource instead of copying.